quisp/modules/QRSA/ConnectionManager/ConnectionManager.cc

```diff
@@ -52,6 +51,9 @@ void ConnectionManager::handleMessage(cMessage *msg) {
 
     int local_qnic_address_to_actual_dst = routing_daemon->return_QNIC_address_to_destAddr(actual_dst);
     auto dst_inf = hardware_monitor->findConnectionInfoByQnicAddr(local_qnic_address_to_actual_dst);
+    if (dst_inf == nullptr) {
+      error("dst inf is null");
```

Comment:
this error message should be more meaningful (e.g. "destination qnic address 'foobar' not found in neighbor_table")
maybe we should put this error check into `HardwareMonitor::findConnectionInfoByQnicAddr`)

Reply:
Ah, right. Let me fix that.